Bottom: dfc3deb2e380ed50923b1b6d30ff23c9c2660c05
Top:    490158a7208d10d874df48ccc2fbf1de8d399913
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-07-25 16:35:58 +0800

Refresh of tun-xdp-support

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 12ff351..1e552fd 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1371,6 +1371,7 @@ static struct sk_buff *tun_build_skb(struct tun_struct *tun,
 
 err_xdp:
 	rcu_read_unlock();
+	this_cpu_inc(tun->pcpu_stats->rx_dropped);
 	return NULL;
 }
